src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.tsx

```diff
@@ -0,0 +1,106 @@
+import 'react-date-range/dist/styles.css';
+import 'react-date-range/dist/theme/default.css';
+
+import {Calendar} from 'react-date-range';
+import React from 'react';
+import moment from 'moment';
+import styled from '@emotion/styled';
+
+import {inputStyles} from 'app/styles/input';
+import DropdownMenu from 'app/components/dropdownMenu';
+import {IconCalendar} from 'app/icons';
+import space from 'app/styles/space';
+
+import InputField from './inputField';
+
+type Props = Omit<InputField['props'], 'field'>;
+
+function handleChangeDate(onChange, onBlur, date, close) {
+  onChange(date, {});
+  onBlur(date, {});
+
+  // close dropdown menu
+  close();
+}
+
+export default function DatePickerField(props: Props) {
+  return (
+    <InputField
+      {...props}
+      field={({onChange, onBlur, value, disabled, id}) => {
+        const dateObj = new Date(value);
+        const inputValue = !isNaN(dateObj.getTime()) ? dateObj : new Date();
+        const dateString = moment(inputValue).format('LL');
+
+        return (
+          <DropdownMenu keepMenuOpen>
+            {({isOpen, getRootProps, getActorProps, getMenuProps, actions}) => (
+              <DatePickerWrapper {...getRootProps()}>
+                <InputWrapper id={id} {...getActorProps()} isOpen={isOpen}>
```

Comment:
This has `name={id}` previously, but it's a div and div's don't have a name attr.

Reply:
Could we end up with duplicate id attributes here?

Reply:
From `getActorProps`? I'm pretty sure those are just on* for the dropdown?

Reply:
No from multiple datepickers being in the dom at the same time.

Reply:
Hmm interesting, could be. If it was an issue already it would be happening independently of this PR.